scripts/test/test_get_accessible_child_nfts.cdc

```diff
@@ -11,34 +12,29 @@ import "MetadataViews"
 /// Assertion method to ensure passing test
 ///
 pub fun assertPassing(result: {Address: {UInt64: MetadataViews.Display}}, expectedAddressToIDs: {Address: [UInt64]}) {
+  for address in expectedAddressToIDs.keys {
+    let expectedIDs: [UInt64] = expectedAddressToIDs[address]!
 
-    for address in result.keys {
-
-        if expectedAddressToIDs[address] == nil {
-            panic("Address ".concat(address.toString()).concat(" found but not expected!"))
-        }
-        let expectedIDs: [UInt64] = expectedAddressToIDs[address]!
-
-        for i, id in result[address]!.keys {
-            if expectedIDs[i] != id {
-              panic("Resulting ID does not match expected ID!")
-            }
-        }
+    for i, id in expectedAddressToIDs[address]! {
+      if result[address]![id] == nil {
+        panic("Resulting ID does not match expected ID!")
+      }
     }
+  }
 }
 
 pub fun main(addr: Address, expectedAddressToIDs: {Address: [UInt64]}){
-  let account = getAuthAccount(addr)
   let manager = getAuthAccount(addr).borrow<&HybridCustody.Manager>(from: HybridCustody.ManagerStoragePath) ?? panic ("manager does not exist")
 
   var typeIdsWithProvider = {} as {Address: [String]} 
   var nftViews = {} as {Address: {UInt64: MetadataViews.Display}} 
 
-  
   let providerType = Type<Capability<&{NonFungibleToken.Provider}>>()
   let collectionType: Type = Type<@{NonFungibleToken.CollectionPublic}>()
 
   // Iterate through child accounts
+  let children = manager.getChildAddresses()
```

Comment:
I think this was meant to be used in the for loop

Reply:
Remnant of logs while debugging why this test wasn't working, I'll remove it